edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/modules/RoutingModule.cs

```diff
@@ -532,6 +532,27 @@ protected override void Load(ContainerBuilder builder)
                 .As<Task<IEdgeHub>>()
                 .SingleInstance();
 
+            // BrokerPropertiesValidator
+            builder.Register(
+                    c =>
+                    {
+                        return new BrokerPropertiesValidator();
```

Comment:
I wasn't sure if I need to register a concrete class or if it is self-discoverable? @vipeller @varunpuranik @richma-ms ?